CMakeLists.txt

```diff
@@ -787,7 +787,7 @@ option(USE_MAINTAINER_MODE
 
 if (USE_MAINTAINER_MODE)
   add_definitions("-DARANGODB_ENABLE_MAINTAINER_MODE=1")
-  if (CMAKE_COMPILER_IS_GNUCC AND NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
+  if (NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
```

Comment:
Question: is `_FORTIFY_SOURCE` understood by MSVC as well?
If guess that even if it's not, setting the define there will not do any harm. Anyone knows?

Reply:
I would hope that jenkins tells me. alas.

Reply:
I just realised that my comment was kinda stupid; of course we can just set some `#define` and if noone picks them up they'll not do harm, really.

CI is not the correct place to blame in this case (only in the sense that setting `_FORTIFY_SOURCE` won't break anything).

So I looked around, and as far as I can tell, MSVC does not support `_FORTIFY_SOURCE`.